src/pybel/ext/__init__.py

```diff
@@ -1,14 +1,47 @@
+"""Extensions for PyBEL can be imported from the :code:`pybel.ext` namespace just like normal modules and packages::
+
+    import pybel.ext.extension
+    # or
+    from pybel.ext import extension as ex
+    # or
+    from pybel.ext.extension import a_function as af
+    # or
+    from pybel.ext.extension import *
+    # or, even
+    from pybel.ext import *
+
+This magic is brought to you by import hooks and :code:`pkg_resources`.
+
+To create your own extension, simply register a :code:`pybel.ext` entry point in your package's :code:`setup.py`::
+
+    setuptools.setup(
+        ...
+        entry_points = {
+            'pybel.ext':
+                ['name_of_your_extension = package.module']
+        }
+        ...
+    )
+
+This works just like the standard :code:`console_scripts` entry point and the syntax follows all the same rules.
+
+Your extension will then be importable as :code:`pybel.ext.name_of_your_extension`
+
+:code:`pybel.ext` exists as a package on its own to trick the Python import system...if we just did this work in PyBEL's :code:`__init__.py`, there could be trouble.
+"""
 from pkg_resources import iter_entry_points
 
 GROUP_NAME = 'pybel.ext'
```

Comment:
Should I do something clever with this line involving `__name__` or something, or just leave it hardcoded as `'pybel.ext'`? If the former, what would the solution be?